Quiz/pages/highscore.html

```diff
@@ -14,7 +14,7 @@
 <body>
   <div class="quiz-container">
     <div id="highScores" class="flex-center flex-column">
-      <h1 id="finalScore">High Scores</h1>
+      <h1 id="finalScore" style="color: black;">High Scores</h1>
       <ul id="highScoresList"></ul>
       <a class="quiz-btn" href="./index.html">Go Home</a>
```

Comment:
Lets fix this as well 
Should be `../../index.html`

Reply:
Fixed!